vcl_templates/www.vcl.erb

```diff
@@ -350,9 +350,8 @@ sub vcl_recv {
   # With the exception of:
   #   - Licensing
   #   - email-alert-frontend (for subscription management)
-  #   - frontend (for sessions across funding registration form)
-  #   - smart answers coronavirus find support flow
-  if (req.url !~ "^/apply-for-a-licence" && req.url !~ "^/email")
+  #   - signon callback
+  if (req.url !~ "^/apply-for-a-licence" && req.url !~ "^/email" && req.url !~ "^/sign-in/callback")
```

Comment:
optional: might be slightly easier to read as a single regex (and similarly for the other one) e.g.
```suggestion
  if (req.url !~ "^/(apply-for-a-licence|email|sign-in/callback)")
```

Reply:
thanks!